Description: do not link against unused libraries
 dpkg-shlibdeps complained about useless dependencies on libgfortran.so.3,
 libpng12.so.0, libX11.so.6 and libm.so.6
Author: Florian Schlichting <fsfs@debian.org>
Forwarded: https://rt.cpan.org/Public/Bug/Display.html?id=95006
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -121,7 +121,6 @@
 
 my $LIBS = join(' ', map { "-l$_" }
 		qw/ cpgplot pgplot /,
-		map { split( ',', $_ ) } @Arg{qw/ XLIB EXLIB /}
 	       );
 
 my $IDIRS = join( ' ', map { "-I$_" } split( ',', $Arg{IDIR} ) );
@@ -203,7 +202,7 @@
 	'pgplot_tmp/libpgplot.a'
 	if -d 'pgplot_tmp';
 
-    $items{LIBS} = [ join( ' ', $LIBDIRS, $LIBS, ExtUtils::F77->runtime ) ];
+    $items{LIBS} = [ join( ' ', $LIBDIRS, $LIBS ) ];
 
     # This is not ideal since it assumes that:
     #    objc is required
